docs/agents/meinberg_syncbox_agent.rst

```diff
@@ -0,0 +1,99 @@
+.. highlight:: rst
+
+.. _meinberg_syncbox:
+
+====================
+Meinberg Syncbox Agent
+====================
+
+The Meinberg Syncbox Agent is an OCS Agent which monitors the Meinberg syncbox, the
+Monitoring is performed via SNMP.
+
+.. argparse::
+    :filename: ../socs/agents/meinberg_syncbox/agent.py
+    :func: add_agent_args
+    :prog: python3 agent.py
+
+Configuration File Examples
+---------------------------
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+OCS Site Config
+```````````````
+
+To configure the Meinberg Syncbox Agent we need to add a MeinbergSyncboxAgent
+block to our ocs configuration file. Here is an example configuration block
+using all of the available arguments::
+
+      {'agent-class': 'MeinbergSyncboxAgent',
+       'instance-id': 'timing-syncbox',
+       'arguments': [['--address', '192.168.2.166'],
+                     ['--port', 161],
+                     ['--mode', 'acq'],
+                     ['--snmp-version', 1]]},
+
+.. note::
+    The ``--address`` argument should be the address of the syncbox on the network.
+    This is not the main Meinberg M1000 device.
+
+Docker Compose
+``````````````
+
+The Meinberg Syncbox Agent should be configured to run in a Docker container. An
+example docker-compose service configuration is shown here::
+
+  ocs-timing-syncbox:
+    image: simonsobs/socs:latest
+    hostname: ocs-docker
+    network_mode: "host"
+    volumes:
+      - ${OCS_CONFIG_DIR}:/config:ro
+    environment:
+      - INSTANCE_ID=timing-syncbox
+      - SITE_HUB=ws://127.0.0.1:8001/ws
+      - SITE_HTTP=http://127.0.0.1:8001/call
+      - LOGLEVEL=info
+
+
+The ``LOGLEVEL`` environment variable can be used to set the log level for
+debugging. The default level is "info".
+
+Description
+-----------
+The Meinberg syncbox synchronizes to GPS and distributes timing mostly over
+the network using PTP.
```

Comment:
The M1000 does this, not the syncbox.

The syncbox syncs from PTP, and distributes signals to attached devices in various formats (IRIG, PPS).